versions.json

```diff
@@ -0,0 +1,3 @@
+[
+  "2.0"
```

Comment:
Might be good to make this match the actual LangSmith semver to avoid confusion? Keep in mind that these pathed sidebar links could be used in the future.

If it won't, we could call this `canary` or something similar instead

Reply:
Or, if we don't expect to share this version of the docs widely, fine to leave

Reply:
We don't use semver for non self hosted langsmith right now so this version specifier is pretty meaningless atm